editor/store/reducer.js

```diff
@@ -34,6 +34,71 @@ import withChangeDetection from '../utils/with-change-detection';
 import { PREFERENCES_DEFAULTS } from './defaults';
 import { insertAt, moveTo } from './array';
 
+/**
+ * The default editor settings
+ *
+ *  alignWide         boolean        Enable/Disable Wide/Full Alignments
+ *  colors            Array          Palette colors
+ *  maxWidth          number         Max width to constraint resizing
+ *  blockTypes        boolean|Array  Allowed block types
+ *  hasFixedToolbar   boolean        Whether or not the editor toolbar is fixed
+ */
+const DEFAULT_SETTINGS = {
```

Comment:
Maybe we should export this constant in './defaults'. Given that we have already have a file for defaults.